gnomad/assessment/validity_checks.py

```diff
@@ -267,12 +270,15 @@ def summarize_variant_filters(
         filters = exp_t.aggregate(hl.agg.counter(exp_t.filters))
         logger.info("Exploded variant filter counts: %s", filters)
 
-    if monoallelic_expr is not None:
-        if isinstance(t, hl.MatrixTable):
-            mono_sites = t.filter_rows(monoallelic_expr).count_rows()
-        else:
-            mono_sites = t.filter(monoallelic_expr).count()
-        logger.info("There are %d monoallelic sites in the dataset.", mono_sites)
+    if site_gt_check_expr is not None:
+        if isinstance(site_gt_check_expr, hl.expr.BooleanExpression):
+            site_gt_check_expr = {"site GT check": site_gt_check_expr}
+        for k, m_expr in site_gt_check_expr.items():
+            if isinstance(t, hl.MatrixTable):
+                gt_check_sites = t.filter_rows(m_expr).count_rows()
+            else:
+                gt_check_sites = t.filter(m_expr).count()
+            logger.info("There are %d %s sites in the dataset.", gt_check_sites, k)
```

Comment:
This is weird now but I didnt know what to rename it "There are %d site GT check sites in the dataset." I almost feel we should drop support for the single expression since this is a breaking change anyways? Require the dict[str, bool expr]?

Reply:
I like that idea (requiring the dict)

Reply:
I doubt many others outside our group are using the validity checks and requiring a dict makes a lot more sense